### ISSUE TITLE
$COLON$COLON links are busted

### ISSUE THREAD

**Issue by zoffixznet** [perl6/doc#1382]
This issue is really part of the link-fixing-refactor the doc site needs; there's a whole bunch of other issues about 404s in search results and other links with some kind of special chars in them.

For this issue, the old style links like https://docs.perl6.org/type/IO$COLON$COLONPath we used to have no longer work. They still exist in the wild, so they should be made to work, to ensure old links on blogs and stuff still work.

I see [some rules in .htaccess](https://github.com/perl6/doc/blob/876afdcd50aa4c5a6f242020fc6a1dc25aca98fe/html/.htaccess#L9-L21) that appear to attempt to make them work, but it seems the rules are outdated and the commented out rules need to be uncommented (and the currently uncommented $COLON rules removed).

Basically both https://docs.perl6.org/type/IO$COLON$COLONPath and https://docs.perl6.org/type/IO::Path URLs are meant to work and link to docs for IO::Path type, in this case.

**Comment by JJ** [comment]
Also related  #561 #1838

**Comment by JJ** [comment]
These rules are there:

    RewriteRule          ^(/?type/.*/[^/]*?)([^/]*?\$COLON[^/]*)$ $1:$2 [N]
    RewriteRule          ^(/?type/.*/[^/]*?)\$COLON([^/\$COLON]*)$       $1:$2 [R=301]

But they do not seem to work. I have no idea why.

**Comment by JJ** [comment]
Ping?

**Comment by JJ** [comment]
.htacess is no longer enabled and this is two years old already. It's been busted for a couple of years, I don't expect anyone to expect they work now. So I'm closing this.
